examples/pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
+  - '*'
```

Comment:
This seems to be the only way for us to get `pnpm` to work again. When inside a workspace, this will only install as another sub-workspace with separate `node_modules`. Otherwise, ignoring examples in the root will just cause the examples to never install at all.